storage/queue.go

```diff
@@ -111,6 +112,96 @@ type PeekMessageResponse struct {
 	MessageText    string `xml:"MessageText"`
 }
 
+// QueueMetadata represents user defined metadata and queue properties on
+// a specific queue.
+//
+// See https://msdn.microsoft.com/en-us/library/azure/dd179384.aspx
+type QueueMetadata struct {
+	ApproximateMessageCount int
+	userDefinedMetadata     map[string]string
+}
+
+// Get retrieve a value for a case insensitive name.
+func (qm *QueueMetadata) Get(name string) (value string) {
+	qm.ensureInit()
+	value, _ = qm.userDefinedMetadata[strings.ToUpper(name)]
```

Comment:
this method will go away anyway but 

1. do not use named returns (like `value string` here)
2. no need to do `val, _ := mapVar[key]`,... you can just do `return mapVar[key]` it works the same :)